src/hooks/useRoleCapabilities/useRoleCapabilities.test.js

```diff
@@ -0,0 +1,118 @@
+import {
+  QueryClient,
+  QueryClientProvider,
+} from 'react-query';
+
+import {
+  act,
+  cleanup,
+  renderHook,
+} from '@folio/jest-config-stripes/testing-library/react';
+import {
+  useOkapiKy,
+  useStripes,
+} from '@folio/stripes/core';
+
+import { useRoleCapabilities } from './useRoleCapabilities';
+import { capabilitiesData } from "../../../test/jest/fixtures";
```

Comment:
I think you're not using regular stripes eslint config, because double quotes should give eslint errors. Please check

Reply:
Ye, thanks, fixed in all files probably